src/rulesets/SparseArrays/array.jl

```diff
@@ -0,0 +1,16 @@
+using SparseArrays
+
+function rrule(::Type{<:SparseMatrixCSC{T,N}}, arr) where {T,N}
+  function SparseMatrix_pullback(Δ)
+    (collect(Δ),)
```

Comment:
Please follow [BlueStyle in always using a `return` keyword on long-form functions](https://github.com/invenia/BlueStyle#method-definitions)

Also, the first return should be the "derivative with respect to the fields of the function itself" https://www.juliadiff.org/ChainRulesCore.jl/dev/#Self-derivative-%CE%94self,-self,-self,-%E1%B9%A1elf-etc
```suggestion
        return NO_FIELDS, collect(Δ)
```
(I think `NO_FIELDS` is fine here for now?)

Reply:
Done